doc/userguide/rules/ipaddr.rst

```diff
@@ -0,0 +1,29 @@
+IP Addresses Match
```

Comment:
I think this needs integration with regular address matching logic in rules. Might confuse ppl otherwise.